<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDlg</class>
 <widget class="QDialog" name="InputDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>190</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>384</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget{
background: #272E3B;
border-radius: 8px;
}

#lbl_title{
font-family : 'Microsoft YaHei';
font-size : 14px;
color : #fff;
}

#edt_name{
	padding-left:16px;
	padding-top:5px;
	padding-bottom:5px;
	background: #1C222D;
	border-radius: 4px;
	font-family : 'Microsoft YaHei';
	font-size : 12px;
	color : #fff;
}

#btn_cancel{
	border:none;
	border-radius:4px;
	font-family : 'Microsoft YaHei';
	font-size : 12px;
	color : #fff;
	background: rgba(255,255,255,0.1);
}

#btn_cancel:pressed{
	background:#4080FF;
}

#btn_confirm{
	border:none;
	border-radius:4px;
	font-family : 'Microsoft YaHei';
	font-size : 12px;
	color : #fff;
	background: #4080FF;
}

#btn_confirm:hover{
	background: #165DFF;
}

#btn_confirm:pressed{
	background: #0E42D2;
}

#btn_confirm:disabled{
	background:#2E4675;
}

#txt_msg{
	font-family:'Microsoft YaHei UI';
	font-size:12px;
	color:#F53F3F;
}

#btn_close{
	background-image:url(:img/close);
	background-position:center;
	background-repeat:no-repeat;
	border:none;
	font-family : 'Microsoft YaHei';
	font-size : 12px;
	color : #fff;
}





</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>24</number>
        </property>
        <property name="rightMargin">
         <number>24</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_title">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_close">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>24</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>24</number>
        </property>
        <item>
         <widget class="LineEditWarp" name="edt_name">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>32</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>请输入用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="txt_msg">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="minimumSize">
           <size>
            <width>56</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_confirm">
          <property name="minimumSize">
           <size>
            <width>56</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>提交</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWarp</class>
   <extends>QLineEdit</extends>
   <header location="global">line_edit_warp.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>InputDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>InputDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_confirm</sender>
   <signal>clicked()</signal>
   <receiver>InputDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
